next/utils/forms.ts

```diff
@@ -541,6 +569,7 @@ export const useFormStepper = (eformSlug: string, eform: EFormValue, callbacks:
       // when we save nextStep, we will validate all (conditional) steps
       validateSteps()
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
probably less hassle with these eslint errors if validateSteps (and other similar) are made into callbacks which don't change with every re-render - and which can then be safely included in the dependency array - note for further rewrite, not necessarily for this PR